src/ng/httpBackend.js

```diff
@@ -46,7 +72,7 @@ function createHttpBackend($browser, createXhr, $browserDefer, callbacks, rawDoc
       });
     } else {
 
-      var xhr = createXhr();
+      var xhr = createXhr(method, url);
```

Comment:
Not for us, but for the benefit of end users who will override the service. Let's say you want to use a custom object for certain urls only..